widget/progressbar.go

```diff
@@ -66,6 +65,18 @@ func (o ProgressBarOptions) WidgetOpts(opts ...WidgetOpt) ProgressBarOpt {
 	}
 }
 
+func (o ProgressBarOptions) Direction(d Direction) ProgressBarOpt {
+	return func(s *ProgressBar) {
+		s.direction = d
+	}
+}
+
+func (o ProgressBarOptions) Inverted(inverted bool) ProgressBarOpt {
+	return func(s *ProgressBar) {
+		s.inverted = inverted
+	}
+}
```

Comment:
I just noticed some other somewhat similar functions in the library that do not take an argument, e.g.
```
func (o DragAndDropOptions) DisableDrag() DragAndDropOpt {
	return func(t *DragAndDrop) {
		t.dragDisabled = true
	}
}
```
Maybe this would be better if it were similar, i.e.
```suggestion
func (o ProgressBarOptions) Invert() ProgressBarOpt {
	return func(s *ProgressBar) {
		s.inverted = true
	}
}
```
?

Reply:
We have examples of both in the codebase.  I prefer the way you currently have it. There are times when its cleaner to pass a variable in rather than call the method or not.